### PR TITLE
docs: fix test cases and add usage in README

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [x] `npm test` passes
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message follows commit guidelines

### CODE REVIEW

**Comment by jtyjty99999** [comment]
+1

File: README.md [hunk 1]
```diff
@@ -45,6 +45,48 @@ exports.redis = {
...
+## Usage
+
+In controller, you can use `app.redis` to get the redis instance, check [ioredis](https://github.com/luin/ioredis#basic-usage) to see how to use.
+
+```js
+// app/controller/home.js
+
+module.exports = app => {
+  return class HomeController
+  extends app.Controller {
```
Comment (atian25) [inline README.md]: 换行了?

**Review by jtyjty99999 — APPROVED** [review]

**Review by atian25 — CHANGES_REQUESTED** [review]
对应的中文文档也加下?

**Comment by jtyjty99999** [comment]
+ dead_horse (egg-redis)

File: README.md (hunk 1 above)
Comment (dead-horse) [reply README.md]: fixed

**Comment by dead-horse** [comment]
写英文的吧，这种简单的就不维护两份了。中文文档我先删掉了，本来也没写

File: test/fixtures/apps/redisapp/app/controller/home.js [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
 
-module.exports = function* () {
-  const a = yield this.redis.get(a);
-
-  this.body = {
-    status: 'success',
-    a,
+module.exports = app => {
+  return class HomeController
+  extends app.Controller {
```
Comment (atian25) [inline test/fixtures/apps/redisapp/app/controller/home.js]: 这里也换行了

File: test/fixtures/apps/redisclusterapp/app/controller/home.js [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
 
-module.exports = function* () {
-  const a = yield this.redis.get(a);
-
-  this.body = {
-    status: 'success',
-    a,
+module.exports = app => {
+  return class HomeController
+  extends app.Controller {
```
Comment (atian25) [inline test/fixtures/apps/redisclusterapp/app/controller/home.js]: 这里

**Review by atian25 — APPROVED** [review]

File: test/fixtures/apps/redisclusterapp/app/controller/home.js (hunk 3 above)
Comment (dead-horse) [reply test/fixtures/apps/redisclusterapp/app/controller/home.js]: 。。。 好了

**Review by atian25 — APPROVED** [review]

**Pull request merged**